pandas/core/series.py

```diff
@@ -252,6 +252,10 @@ def _constructor_expanddim(self):
         from pandas.core.frame import DataFrame
         return DataFrame
 
+    @property
```

Comment:
I did this so constructor_sliced could be used could for functions that return a series instance but take a dataframe or series as input. Otherwise I would need to check for input type. At least this was my reasoning. :-)